modules/oc-by-example-content.adoc

```diff
@@ -0,0 +1,4723 @@
+// NOTE: The contents of this file are auto-generated
+
+[id="openshift-cli_{context}"]
+= OpenShift CLI (oc) commands
+
+
+
+ifdef::openshift-enterprise,openshift-origin[] 
+
+== oc adm build-chain
+Output the inputs and dependencies of your builds
+
+.Example usage
+[source,terminal,options="nowrap"]
+----
+  # Build the dependency tree for the 'latest' tag in <image-stream>
```

Comment:
There's extra leading space for these; I've never done this, but maybe that doesn't matter to asciidoctor.